frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetToRemoteAction.java

```diff
@@ -160,8 +160,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
               remoteTrackingBranch.getName(),
               currentCommitSha));
 
-      dialogBuilder.yesText(getString("action.GitMachete.BaseResetToRemoteAction.info-dialog.ok-text"))
```

Comment:
Okay... so how do buttons look like now as compared to what was previously?

Reply:
previous:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/19799111/195891247-e9abb293-693a-405c-91f1-98eeec60d9df.png">

new (note it is scrollable):
<img width="477" alt="image" src="https://user-images.githubusercontent.com/19799111/195891592-81b88677-f8a0-4819-8b0b-d8375453e8ae.png">
<img width="467" alt="image" src="https://user-images.githubusercontent.com/19799111/195891642-0c1c2c0e-822b-49b3-aa60-123d159c3c81.png">